components/table/table-wrapper.js

```diff
@@ -243,11 +245,14 @@ export class TableWrapper extends RtlMixin(PageableMixin(SelectionMixin(LitEleme
 				--d2l-table-border-radius: 0.3rem;
 				--d2l-table-border-radius-sticky-offset: calc(1px - var(--d2l-table-border-radius));
 				--d2l-table-cell-height: 41px; /* min-height to be 62px including border */
-				--d2l-table-cell-padding: 0.5rem 1rem;
-				--d2l-table-cell-padding-alt: calc(0.5rem - 1px) 1rem 0.5rem 1rem;
+				--d2l-table-cell-padding: 0.5rem 0.75rem;
+				--d2l-table-cell-padding-alt: calc(0.5rem - 1px) 0.75rem 0.5rem 0.75rem;
```

Comment:
One other thing to confirm with Nicole is that this also changes the end padding so we'd just want to confirm that that should now also be decreased everywhere.